turbinia/turbiniactl.py

```diff
@@ -750,63 +792,140 @@ def main():
+  elif args.command == 'googleclouddisk':
+    evidence_ = evidence.GoogleCloudDisk(
+        name=name, disk_name=disk_name, project=project, zone=zone,
+        source=source)
+  elif args.command == 'googleclouddiskembedded':
+    parent_evidence_ = evidence.GoogleCloudDisk(
+        name=name, disk_name=disk_name, project=project, source=source,
+        mount_partition=mount_partition, zone=zone)
+    evidence_ = evidence.GoogleCloudDiskRawEmbedded(
+        name=name, disk_name=disk_name, project=project, zone=zone,
+        embedded_path=embedded_path)
+    evidence_.set_parent(parent_evidence_)
+  elif args.command == 'hindsight':
+    if format not in ['xlsx', 'sqlite', 'jsonl']:
+      msg = 'Invalid output format.'
+      raise TurbiniaException(msg)
+    if browser_type not in ['Chrome', 'Brave']:
+      msg = 'Browser type not supported.'
+      raise TurbiniaException(msg)
+    source_path = os.path.abspath(source_path)
+    evidence_ = evidence.ChromiumProfile(
+        name=name, source_path=source_path, output_format=format,
+        browser_type=browser_type)
+  elif args.command == 'rawmemory':
+    source_path = os.path.abspath(source_path)
+    evidence_ = evidence.RawMemory(
+        name=name, source_path=source_path, profile=profile,
+        module_list=args.module_list)
+
   if evidence_ and not args.force_evidence:
-    if config.SHARED_FILESYSTEM and evidence_.cloud_only:
-      log.error(
-          'The evidence type {0:s} is Cloud only, and this instance of '
-          'Turbinia is not a cloud instance.'.format(evidence_.type))
-      sys.exit(1)
-    elif not config.SHARED_FILESYSTEM and evidence_.copyable:
+    if not config.SHARED_FILESYSTEM and evidence_.copyable:
       if os.path.exists(evidence_.local_path):
         output_manager = OutputManager()
         output_manager.setup(evidence_.type, request_id, remote_only=True)
         output_manager.save_evidence(evidence_)
       else:
-        log.error(
+        msg = (
             'The evidence local path does not exist: {0:s}. Please submit '
             'a new Request with a valid path.'.format(evidence_.local_path))
-        sys.exit(1)
+        raise TurbiniaException(msg)
     elif not config.SHARED_FILESYSTEM and not evidence_.cloud_only:
-      log.error(
+      msg = (
           'The evidence type {0:s} cannot run on Cloud instances of '
           'Turbinia. Consider wrapping it in a '
           'GoogleCloudDiskRawEmbedded or other Cloud compatible '
           'object'.format(evidence_.type))
-      sys.exit(1)
+      raise TurbiniaException(msg)
 
-  # If we have evidence to process and we also want to run as a server, then
-  # we'll just process the evidence directly rather than send it through the
-  # PubSub frontend interface.  If we're not running as a server then we will
-  # create a new TurbiniaRequest and send it over PubSub.
   request = None
-  if evidence_ and args.server:
```

Comment:
Im not sure if I understand this right. There's only one reference to server flag in the code and that is to start the logger.

Reply:
If we remove this (since it's not being used) we should also remove it from `turbinia/docker/server/start.sh` and any other e2e tests  that might still reference that.

Reply:
Fixed